### PR TITLE
Feat/lockdrop cl pools migration #NTRN-515

### PR DESCRIPTION
**Pull request by ratik**
* [Task](https://p2pvalidator.atlassian.net/browse/NTRN-515)
* [Contracts](https://github.com/neutron-org/neutron-tge-contracts/pull/51)
* [Run](https://github.com/neutron-org/neutron-tests/actions/runs/5763717576)

### CODE REVIEW

File: src/testcases/run_in_band/tge.auction_migration.test.ts [hunk 1]
```diff
@@ -0,0 +1,1876 @@
...
+          },
+        }),
+        [],
+        {
+          gas_limit: Long.fromString('8000000'),
+          amount: [{ denom: tge.chain.denom, amount: '20000' }],
+        },
+      );
+    });
+    it('should migrate users', async () => {
```
Comment (sotnikov-s) [inline src/testcases/run_in_band/tge.auction_migration.test.ts:1493]: let's add pagination usage? I mean with a limit parameter usage and check whether it workes i.e. users are migrated in batches

File: src/testcases/run_in_band/tge.auction_migration.test.ts [hunk 2]
```diff
@@ -0,0 +1,1876 @@
...
+        ),
+      ).rejects.toThrowError(/Migration is not in MigrateUsers state/);
+    });
+    it('should migrate liquidity', async () => {
+      await cmInstantiator.executeContract(
+        tge.contracts.lockdrop,
+        JSON.stringify({
+          migrate_from_xyk_to_cl: {
+            migrate_liquidity: {
+              slippage_tolerance: '0.05',
```
Comment (sotnikov-s) [inline src/testcases/run_in_band/tge.auction_migration.test.ts:1482]: please add a check that a slippage tolerance greater than the max one results in a failure

File: src/testcases/run_in_band/tge.auction_migration.test.ts [hunk 3]
```diff
@@ -0,0 +1,1876 @@
...
+          tge.contracts.lockdrop,
+          JSON.stringify({
+            migrate_from_xyk_to_cl: {
+              migrate_users: {},
+            },
+          }),
+        ),
+      ).rejects.toThrowError(/Migration is not in MigrateUsers state/);
+    });
+    it('should migrate liquidity', async () => {
```
Comment (sotnikov-s) [inline src/testcases/run_in_band/tge.auction_migration.test.ts:1476]: please add at least a rough check of lockdrop contract's lp token balances before liquidity migration and after it, e.g. before XYK != 0, CL == 0, after XYK == 0, CL != 0 or something

**Review by sotnikov-s — APPROVED** [review]

**Comment by ratik** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/5453471110

**Comment by sotnikov-s** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6301303901

**Comment by sotnikov-s** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6308640997

**Pull request closed without merging**